src/customizations/volto-form-block/components/Sidebar.jsx

```diff
@@ -0,0 +1,325 @@
+// CUSTOMIZATION:
+// - 112-113 reset subscription limit to default when set_limit is not active
+
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { useSelector, useDispatch } from 'react-redux';
+import {
+  Segment,
+  Accordion,
+  Form,
+  Button,
+  Grid,
+  Confirm,
+  Dimmer,
+  Loader,
+} from 'semantic-ui-react';
+import { defineMessages, useIntl, FormattedMessage } from 'react-intl';
+
+import { Icon } from '@plone/volto/components';
+
+import upSVG from '@plone/volto/icons/up-key.svg';
+import downSVG from '@plone/volto/icons/down-key.svg';
+import downloadSVG from '@plone/volto/icons/download.svg';
+import deleteSVG from '@plone/volto/icons/delete.svg';
+
+import warningSVG from '@plone/volto/icons/warning.svg';
+
+import {
+  getFormData,
+  exportCsvFormData,
+  clearFormData,
+} from 'volto-form-block/actions';
+
+import { BlockDataForm } from '@plone/volto/components';
+import { flattenToAppURL } from '@plone/volto/helpers';
+import { getFieldName } from 'volto-form-block/components/utils';
+
+import './Sidebar.css';
```

Comment:
Nel commento al file css sopra mi riferisco a questo import che può diventare così

```suggestion
import 'volto-form-block/components/Sidebar.css';
```